### PR TITLE
label_sync: add wg/component-standard

### PR DESCRIPTION
**Pull request by nikhita**
Ref: https://github.com/kubernetes/community/pull/3008#discussion_r246664028

/hold
until https://github.com/kubernetes/community/pull/3008 is merged

/cc @luxas @sttts @mtaufen 
/assign @cblecker

### CODE REVIEW

File: label_sync/labels.yaml [hunk 1]
```diff
@@ -577,6 +577,13 @@ default:
       target: both
       prowPlugin: label
       addedBy: anyone
+    - color: ee9900
```
Comment (luxas) [inline label_sync/labels.yaml]: Should this be here?

Comment (nikhita) [reply label_sync/labels.yaml]: gah, bad copy paste. fixed.

**Comment by nikhita** [comment]
/assign @Katharine 
for approval

**Comment by nikhita** [comment]
Cancelling the hold since https://github.com/kubernetes/community/pull/3008 merged.

/hold cancel

**Comment by Katharine** [comment]
I wonder if it'd make sense to make labels.css be generated by bazel at build time. In any case:

/approve
/pony lgtm

**Pull request merged**